web/styles/zulip.css

```diff
@@ -611,8 +611,9 @@ body.has-overlay-scrollbar {
         justify-content: space-evenly;
```

Comment:
Let's squash CSS changes in 2nd, 3rd, 4th and 5th commit together since it doesn't make sense to keep them apart to try to make sense.